@sanity/desk-tool v2.28.0 crashes sanity-plugin-intl-input due to renamed `ConfirmDelete` component

Upgrading to `@sanity/desk-tool` v2.28.0 triggers a crash in the `sanity-plugin-intl-input` with the following error:

```
Module not found: Error: Can't resolve '@sanity/desk-tool/lib/components/ConfirmDelete' in
'[...]/node_modules/sanity-plugin-intl-input/lib/actions'
```

The `[...]/components/ConfirmDelete` module seems to have been renamed to `[...]/components/confirmDeleteDialog` in this commit: https://github.com/sanity-io/sanity/commit/02017a06e04c9c10780fe406859946866f9ebdb4. It looks like the `ConfirmDelete` component has also been renamed to `ConfirmDeleteDialog`.

The import of the `ConfirmDeleteModule` in `DeleteWithi18nAction.tsx` should be updated to reflect the renamed module and the `ConfirmDeleteModule?.ConfirmDelete` probably changed to `ConfirmDeleteModule?.ConfirmDeleteDialog` (this is just what I can read from the source code changes – I have not tested this yet).

--- a/src/actions/DeleteWithi18nAction.tsx
+++ b/src/actions/DeleteWithi18nAction.tsx
@@ -1,6 +1,6 @@
 import React from 'react';
 import TrashIcon from 'part:@sanity/base/trash-icon'
-import * as ConfirmDeleteModule from '@sanity/desk-tool/lib/components/ConfirmDelete';
+import * as ConfirmDeleteModule from '@sanity/desk-tool/lib/components/confirmDeleteDialog';
 import { IResolverProps, IUseDocumentOperationResult } from '../types';
 import { getConfig, getSanityClient, getBaseIdFromId, getTranslationsFor } from '../utils';
 import { useDocumentOperation } from '@sanity/react-hooks';
@@ -16,8 +16,8 @@ const DISABLED_REASON_TITLE = {
 
 export const DeleteWithi18nAction = ({ id, type, draft, published, onComplete }: IResolverProps) => {
   const toast = useToast();
-  const ConfirmDelete = React.useMemo(() => (
-    ConfirmDeleteModule?.ConfirmDelete ?? ConfirmDeleteModule?.default
+  const ConfirmDeleteDialog = React.useMemo(() => (
+    ConfirmDeleteModule?.ConfirmDeleteDialog ?? ConfirmDeleteModule?.default
   ), [ConfirmDeleteModule]);
   const config = React.useMemo(() => getConfig(type), [type]);
   const baseDocumentId = React.useMemo(() => getBaseIdFromId(id), [id]);
@@ -68,17 +68,17 @@ export const DeleteWithi18nAction = ({ id, type, draft, published, onComplete }:
   const dialogContent = React.useMemo(() => {
     if (isConfirmDialogOpen) {
       return (
-        <ConfirmDelete
-          draft={draft}
-          published={published}
+        <ConfirmDeleteDialog
+          action='delete'
+          id={draft?._id || id}
+          type={type}
           onCancel={onDialogCancel}
           onConfirm={onDialogConfirm}
         />
       )
     }
-
     return null;
-  }, [isConfirmDialogOpen, draft, published, onDialogCancel, onDialogConfirm]);
+  }, [isConfirmDialogOpen, draft, id, type, onDialogCancel, onDialogConfirm]);
 
   return {
     onHandle,

